gridscale/datasource_gridscale_loadbalancer.go

```diff
@@ -83,6 +83,10 @@ func dataSourceGridscaleLoadBalancer() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"proxy_protocol": {
```

Comment:
is not set in?
```
if err = d.Set("backend_server", flattenLoadbalancerBackendServers(loadbalancer.Properties.BackendServers)); err != nil {
		return fmt.Errorf("%s error setting BackendServers: %v", errorPrefix, err)
	}
```
otherwise how the test is passed.